From 69305cc603c197dafadc441cd125d63500c565df Mon Sep 17 00:00:00 2001
From: Dongwon Kim <dongwon.kim@intel.com>
Date: Thu, 1 Dec 2022 14:19:22 -0800
Subject: [PATCH 15/34] ui/gtk-egl: should assume dmabuf is unusable once
 rendered

Shouldn't try to recreate texture out of the submitted dmabuf in refresh
routine because it could be invalid anytime after being rendered.

Signed-off-by: Dongwon Kim <dongwon.kim@intel.com>
---
 ui/gtk-egl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/ui/gtk-egl.c b/ui/gtk-egl.c
index a204d235b3..01a2ae7297 100644
--- a/ui/gtk-egl.c
+++ b/ui/gtk-egl.c
@@ -162,7 +162,6 @@ void gd_egl_refresh(DisplayChangeListener *dcl)
 #ifdef CONFIG_GBM
         if (vc->gfx.guest_fb.dmabuf) {
             egl_dmabuf_release_texture(vc->gfx.guest_fb.dmabuf);
-            gd_egl_scanout_dmabuf(dcl, vc->gfx.guest_fb.dmabuf);
         }
 #endif
     }
-- 
2.30.2

